src/Tickets/Commerce/Settings.php

```diff
@@ -271,8 +271,9 @@ public function get_settings() {
 						'</strong>'
 					),
 					Completed::SLUG => sprintf(
-					// Translators: %1$s: The word "ticket" in lowercase. %2$s: `<strong>` opening tag. %3$s: `</strong>` closing tag.
-						esc_html__( 'Only decrease available %1$s stock if an order is confirmed as %2$sCompleted%3$s by the payment gateway.', 'event-tickets' ),
+					// Translators: %1$s: The word "ticket" in lowercase. %2$s: The word "ticket" in lowercase. %3$s: `<strong>` opening tag. %4$s: `</strong>` closing tag.
```

Comment:
We can keep just one placeholder for the word "ticket" in lowercase

Reply:
Updated.